frontend/test/metabase/scenarios/sharing/public.cy.spec.js

```diff
@@ -201,4 +201,106 @@ describe("scenarios > public", () => {
       }),
     );
   });
+
+  describe("public dashboard with parameters linked to a custom field (metabase#25473)", () => {
```

Comment:
Mostly copy the setup from https://github.com/metabase/metabase/blob/master/frontend/test/metabase/scenarios/embedding/embedding-dashboard.cy.spec.js